docker-local/library/prometheus/playbooks/templates/prometheus.yaml.tmpl

```diff
@@ -35,4 +35,4 @@ scrape_configs:
   - job_name: 'consul-registered-skydive-exporters'
     consul_sd_configs:
       - server: "{{ consul_endpoint }}"
-        services: [consul-registered-skydive-exporters]
+        services: [consul-registered-edge-exporters]
```

Comment:
This is my fault, [consul-registered-skydive-exporters] (line previous to merge) is the correct one, since is the name Kalman uses to register the Skydive exporter, this change comes from an error in this line from my private fork (I insist, [consul-registered-edge-exporters] is wrong).